workflow_scripts/check_model.py

```diff
@@ -38,6 +45,8 @@ def run_backend_ort(model_path, test_data_set=None):
         onnxruntime.InferenceSession(model_path)
         ort_test_dir_utils.create_test_dir(model_path, './', test_utils.TEST_ORT_DIR)
         ort_test_dir_utils.run_test_dir(test_utils.TEST_ORT_DIR)
+        os.remove(tar_gz_path)
```

Comment:
Why is this needed? Shouldn't `open` in mode `"w"` just overwrite it?

Reply:
Good point. I have removed it.